tests/ci/run_benchmark_build_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+install_dir="$(pwd)/../librypto_install_dir"
+openssl_url='https://github.com/openssl/openssl.git'
+openssl_1_1_branch='OpenSSL_1_1_1-stable'
+openssl_3_0_branch='openssl-3.0'
+
+function build_openssl_1_1 {
+echo "building OpenSSL 1.1"
+git clone --branch "${openssl_1_1_branch}" "${openssl_url}" "../openssl-1.1"
+pushd "../openssl-1.1"
+mkdir -p "${install_dir}/openssl-1.1"
+./config --prefix="${install_dir}/openssl-1.1" --openssldir="${install_dir}/openssl-1.1"
+make
```

Comment:
To speed this up please specify the number of parallel jobs:
```suggestion
make "-j${NUM_CPU_THREADS}"
```